Doc/includes/sqlite3/row_factory.py

```diff
@@ -11,3 +11,5 @@ def dict_factory(cursor, row):
 cur = con.cursor()
 cur.execute("select 1 as a")
 print(cur.fetchone()["a"])
+
```

Comment:
Nit: Newline is not needed since the snippert is short and there is no logical grouping, e.g.:

```py
con = sqlite3.connect(":memory:")
con.row_factory = dict_factory

cur = con.cursor()
cur.execute("select 1 as a")

print(cur.fetchone()["a"])
```